### PR TITLE
Create maven.yml

### PR DESCRIPTION
**Pull request by Domene99**
Setup testing workflow for week 5. I'm not sure if this will work but I think it will

### CODE REVIEW

**Comment by Domene99** [comment]
Great it works I just don't know why the check is called build and not test

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Test with Maven
+      run: mvn test --file walkthroughs/week-5-tdd/project/pom.xml
```
Comment (ssmall) [inline .github/workflows/maven.yml]: nit: you can add `-B` and `-ntp` to make the build logs a bit cleaner

File: .github/workflows/maven.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
```
Comment (ssmall) [inline .github/workflows/maven.yml]: This is the name of the job, so if you want it to show up as `test`, you can change it here.

**Review by ssmall — APPROVED** [review]
Looks great!

File: .github/workflows/maven.yml (hunk 1 above)
Comment (Domene99) [reply .github/workflows/maven.yml]: -ntp is not a flag I could find

**Pull request merged**